.ebextensions/npm.config

```diff
@@ -39,6 +39,6 @@ container_commands:
   "15_NpmInstall":
     command: "export PATH=$PATH:/usr/bin:$NODE_HOME/bin && $NODE_HOME/bin/node $NODE_HOME/bin/npm install --unsafe-perm=true --allow-root --scripts-prepend-node-path=true"
   "16_NpmBuildAgoraClient":
-    command: "export PATH=$PATH:/usr/bin:$NODE_HOME/bin && $NODE_HOME/bin/node $NODE_HOME/bin/npm run --scripts-prepend-node-path=true build:aot"
+    command: "export PATH=$PATH:/usr/bin:$NODE_HOME/bin && if [ $APP_ENV = "develop" ]; then $NODE_HOME/bin/node $NODE_HOME/bin/npm run --scripts-prepend-node-path=true build:dev; else $NODE_HOME/bin/node $NODE_HOME/bin/npm run --scripts-prepend-node-path=true build:aot; fi"
   "17_NpmBuildAgoraServer":
     command: "export PATH=$PATH:/usr/bin:$NODE_HOME/bin && $NODE_HOME/bin/node $NODE_HOME/bin/npm run --scripts-prepend-node-path=true build:server:prod"
```

Comment:
We need a condition here just like the above if. This time we need three `if` conditions ```build:server:dev```, ```build:server:staging```, and ```build:server:prod```.